### DISCUSSION TITLE
Verifica webhook

### DISCUSSION THREAD

**Discussion by tramic** [fattureincloud/api#485] [Q&A] [answered]
Buongiorno, 
premetto che ho guardato le altre discussioni prima di scriverne una nuova.
Ho un problema con la verifica di una webhook subscription
Il target della mia subscription è uno script in PHP (es https://yyyyyyy.com/webhook.php)
Utilizzando l'SDK PHP creo correttamente il webhook
{
      "id": "SUB1232",
      "sink": "https://yyyyyyy.com/webhook.php",
      "verified": false,
      "types": [],
      "config": {
        "mapping": "binary"
      }
  }

Ho provato ad inserire all'interno di webhook.php il seguente script 
`
header("Content-type: application/json; charset=utf-8");
$headers = apache_request_headers();
$array = array("verification" => $headers["x-fic-verification-challenge"]);
echo json_encode($array,JSON_PRETTY_PRINT);
`

Solo che non funziona, ovvero il campo verified non va a true, mi potete aiutare o capire dove sbaglio?

Grazie

**Comment by valmoz** [comment, marked as answer]
Buongiorno @tramic ,
le suggerisco di provare ad effettuare una chiamata verso il suo endpoint usando Postman o simili, per provare a verificare che l'evento di verifica venga gestito correttamente dal suo codice.

Restiamo a disposizione per ulteriori chiarimenti.
Grazie
Mauro

**Comment by tramic** [comment]
Grazie mille 
Funziona correttamente
